style/ruby/.rubocop.yml

```diff
@@ -467,6 +467,9 @@ Style/InitialIndentation:
   Description: >-
     Checks the indentation of the first non-blank non-comment line in a file.
   Enabled: false
+  
+Style/SymbolArray:
```

Comment:
@awead but that is not the official description for the Linter.  Where are those coming from?  From what I can see your descriptions do not match the comments in the code.  I do not think making stuff up is wise.